app/code/Magento/Eav/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessor.php

```diff
@@ -64,20 +64,55 @@ private function addFilterGroupToCollection(
         AbstractDb $collection
     ) {
         $fields = [];
+        $customFilters = [];
         foreach ($filterGroup->getFilters() as $filter) {
             $isApplied = false;
+            $applyLater = false;
```

Comment:
same like here `$isApplied = $applyLater = false;`

Reply:
Sorry, but I cannot agree with you